example-events-client/utils.py

```diff
@@ -0,0 +1,171 @@
+import requests
+from typing import Optional
+
+
+def print_streaming_events(
+    events: list[dict],
+) -> None:
+    """Print all `CompletionChunkEvent` tokens.
+
+    Args:
+        events: The events to print.
+        printed_event_ids: The set of ids of already printed events. Used to prevent duplicate printing.
+    Returns:
+        The updated set of printed events.
+    """
```

Comment:
Args don't match. I think we should get rid of this function anyway and do a simple match on the event types coming through.